pkg/windows/windows.go

```diff
@@ -467,7 +467,24 @@ func (vm *windows) ConfigureWICD(apiServerURL, watchNamespace string, credential
 	}
 	wicdServiceArgs := fmt.Sprintf("controller --windows-service --log-dir %s --api-server %s --sa-ca %s --sa-token %s --namespace %s",
 		wicdLogDir, apiServerURL, wicdCAFile, wicdTokenFile, watchNamespace)
-	wicdService, err := newService(wicdPath, WicdServiceName, wicdServiceArgs, nil)
+
+	// if WICD crashes, attempt to restart WICD after 30 seconds, 60 seconds, and then every 2 minutes after that.
```

Comment:
nit: does starting at 30s for WICD seem like a slow reaction? For comparison, k8s exp backoff for containers starts at 10s